apple/user_access.md

```diff
@@ -0,0 +1,19 @@
+# Apple Account User Access
```

Comment:
Should we link to https://mana.mozilla.org/wiki/display/RelEng/Apple+App+Store+access ?